spec/support/geocoder_stubs.rb

```diff
@@ -0,0 +1,272 @@
+# Geocode information so that Geocoder can return the information needed without having to hit the API
+# @see https://github.com/alexreisner/geocoder#testing-apps-that-use-geocoder
+
+
+# If  you use an address in the tests, YOU MUST CREATE THE RESULT THAT GEOCODER WILL RETURN for it.
+# That means you must create and entry for it in the 'stubbed_addresses' hash below.
+#
+# Unless you are specifically testing the results, it doesn't really matter what data it returns just as
+# long as it is valid. So just copy the results (the hash) returned from one of the addresses here and
+# use that for the address you are creating.
+#
+
+
+# address info and coordinates with just the street number changed
+#  The company factory will create this address, incrementing the address number
+#   so instead of hardcoding a lot of these, we just use this method
+def hunforetagarevagen(address_num)
+  {
+      'latitude' => 56.7422437,
+      'longitude' => 12.7206453,
+      'address' => "Hundforetagarevägen #{address_num}, 310 40, Harplinge, Sverige",
+      'city' => 'Harplinge',
+      'state' => 'Hallands län',
+      'postal_code' => '310 40',
+      'country' => 'Sverige',
+      'country_code' => 'SE'
+  }
+end
+
+
+stubbed_addresses = {
+
+    "Hundvägen 101, 310 40, Harplinge, Sverige" =>
+        {
+            'latitude' => 56.7422437,
+            'longitude' => 12.7206453,
+            'address' => 'Hundvägen 101, 310 40, Harplinge, Sverige',
+            'city' => 'Harplinge',
+            'state' => 'Hallands län',
+            'postal_code' => '310 40',
+            'country' => 'Sverige',
+            'country_code' => 'SE'
+        },
+
+
+    "Hundforetagarevägen 1, 310 40, Harplinge, Sverige" =>
+        hunforetagarevagen(1),
+
+    "Hundforetagarevägen 2, 310 40, Harplinge, Sverige" =>
+        hunforetagarevagen(2),
+
+    "Hundforetagarevägen 3, 310 40, Harplinge, Sverige" =>
+        hunforetagarevagen(3),
+
+    "Hundforetagarevägen 4, 310 40, Harplinge, Sverige" =>
+        hunforetagarevagen(4),
+
+    "Hundforetagarevägen 5, 310 40, Harplinge, Sverige" =>
+        hunforetagarevagen(5),
+    "Hundforetagarevägen 5, 310 40, Harplinge, Sverige" =>
+        hunforetagarevagen(5),
+
+
+    "Hundforetagarevägen 1, 310 40, HasRegionBorg, Sverige" =>
+        {
+            'latitude' => 56.7422437,
+            'longitude' => 12.7206453,
+            'address' => 'Hundvägen 101, 310 40, HasRegionBorg, Sverige',
+            'city' => 'HasRegionBorg',
+            'state' => 'Hallands län',
+            'postal_code' => '310 40',
+            'country' => 'Sverige',
+            'country_code' => 'SE'
+        },
+
+
+    "Hundforetagarevägen 1, 310 40, NoRegionBorg, Sverige" =>
+        {
+            'latitude' => 56.7422437,
+            'longitude' => 12.7206453,
+            'address' => 'Hundvägen 101, 310 40, HasRegionBorg, Sverige',
+            'city' => 'HasRegionBorg',
```

Comment:
This one (and all of them) is used somewhere in the tests.  IOW, there's a test that creates an address like this.  (I think it's the same test that defines `NoRegionBorg`).  So it's needed.

But I'm glad you're looking carefully at these -- you've caught my sloppy duplicates! :-) 👍